tests/js/spec/components/todoList.spec.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import TodoList from 'app/components/onboardingWizard/todos';
```

Comment:
Can we rename `todos` --> `todoList` to match (and can we move this test file to match the component path e.g. `tests/js/spec/components/onboardingWizard`?